### ISSUE TITLE
Running with --interval and an invalid duration leads to nil pointer dereference

### ISSUE THREAD

**Issue by AlexanderThaller** [sachaos/viddy#66]
```
» ./viddy --version
viddy version: 0.3.6
```

```
» ./viddy -n asdasd
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x40 pc=0x7df725]

goroutine 1 [running]:
main.main()
	/home/runner/work/viddy/viddy/main.go:43 +0x1c5
```

**Comment by sachaos** [comment]
This problem have been solved by viddy v1.0.0 RC.
https://github.com/sachaos/viddy/issues/117

**Comment by sachaos** [comment]
Viddy v1.0.0 is available now.
